src/DependencyInjection/SyliusRefundExtension.php

```diff
@@ -48,7 +48,10 @@ public function prepend(ContainerBuilder $container): void
 
         $container->prependExtensionConfig('sylius_labs_doctrine_migrations_extra', [
             'migrations' => [
-                'Sylius\RefundPlugin\Migrations' => ['Sylius\Bundle\CoreBundle\Migrations'],
+                'Sylius\RefundPlugin\Migrations' => [
+                    'Sylius\Bundle\CoreBundle\Migrations',
+                    'Sylius\Bundle\AdminApiBundle\Migrations',
+                ],
```

Comment:
Maybe we should add functional test for this?

Reply:
We could, but not in this PR imo